spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -215,6 +215,84 @@ definitions:
             fill: SubSystemReport
             desc: Reported status of individual subsystems
 
+ - StatusJournalItem:
+    short_desc: Sub-system Status report
+    embedded_type: true
+    desc: >
+      Report the general and specific state of a sub-system.  If the generic
+      state is reported as initializing, the specific state should be ignored.
+    fields:
+        - uptime:
+            type: u32
+            desc: Milliseconds since system startup
+        - component:
+            type: u16
+            desc: Identity of reporting subsystem
+            fields:
+              - 0-15:
+                  desc: Subsystem
+                  values:
+                      - 0: Primary GNSS Antenna
+                      - 1: Measurement Engine
+                      - 2: Corrections Client
+                      - 3: Differential GNSS Engine
+                      - 4: CAN
+                      - 5: Wheel Odometry
+                      - 6: Sensor Fusion Engine
+        - generic:
+            type: u8
+            desc: Generic form status report
+            fields:
+              - 0-7:
+                  desc: Generic
+                  values:
+                      - 0: OK/Nominal
+                      - 1: Initializing
+                      - 2: Unknown
+                      - 3: Degraded
+                      - 4: Unusable
+        - specific:
+            type: u8
+            desc: Subsystem specific status code
+
+ - MSG_STATUS_JOURNAL:
+    id: 0xFFFD
+    short_desc: Status report journal
+    desc: >
+        The status journal message contains past status reports (see MSG_STATUS_REPORT)
+        and functions as a error/event storage for telemetry purposes.
+    fields:
+        - reporting_system:
+            type: u16
+            desc: Identity of reporting system
+            fields:
+              - 0-15:
+                 desc: System
+                 values:
+                     - 0: Starling
+                     - 1: Precision GNSS Module (PGM)
+        - sbp_version:
+            type: u16
+            desc: SBP protocol version
+            fields:
+              - 8-15:
+                  desc: SBP major protocol version number
+              - 0-7:
+                  desc: SBP minor protocol version number
+        - n_status_reports:
```

Comment:
Based on discussions with @reimerix, it is suppose to be the "count of status reports", this was a misunderstanding on my part. I've since resolved my comment from [here](https://github.com/swift-nav/libsbp/pull/1123#discussion_r859257973).

Reply:
ok